angular/src/lib/activity-button/activity-button.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+export type ActivityButtonSize = 56 | 68 | 84;
+export type ActivityButtonType = 'chat' | 'camera' | 'contact-card' | 'meetings' | 'whiteboard' | 'files' | 'share-screen' | 'tasks' | '';
+
+@Component({
+  selector: 'md-activity-button',
+  template: `
+  <button
+    [ngClass]='btnNgClass'
```

Comment:
In another PR it was mentioned that we don't abbreviate button in our new libraries

Reply:
Let's discuss this during standup.  We've recently allowed button to be abbreviated as a prop.

Reply:
I can change the name btnNgClass to buttonNgClass. Thanks for keep update.